Modules/TRD/include/TRD/PulseHeightTrackMatch.h

```diff
@@ -69,6 +69,9 @@ class PulseHeightTrackMatch final : public TaskInterface
   std::shared_ptr<TH1F> mParsingTimePerTF;
   std::shared_ptr<TH1F> mTracksPerEvent;
   std::shared_ptr<TH1F> mTrackletsPerMatchedTrack;
+  std::shared_ptr<TH1F> mDebugCounter;
```

Comment:
That's unusual, but if it works, then sure

Reply:
I just want to be able to see live when things go wrong, as with the data I have been looking at the updates to the tasks main histogram is incredibly slow. It seemed like the easiest way of seeing where it is rejecting the data. This will ultimately come out of course, later on.